Tasks/XamarinTestCloud/task.json

```diff
@@ -110,10 +110,19 @@
                 "ja_JP": "Japanese (Japan)",
                 "ru_RU": "Russian (Russia)",
                 "es_MX": "Spanish (Mexico)",
-                "es_ES": "Spanish (Spain)"
+                "es_ES": "Spanish (Spain)",
+                "user": "User Defined"
             }
         },
         {
+            "name": "userDefinedLocale",
+            "type": "string",
+            "label": "User Defined Locale",
+            "groupName": "advanced",
+            "defaultValue": "",
+            "required": false
```

Comment:
Can we please make the "userDefinedLocale" field visible only when the "locale" field is set to "User Defined"?  This can be done by adding the following after the "userDefinedLocale" field's "required":false property:
        "visibleRule": "locale = user"